roles/python/tasks/install_dependencies.yml

```diff
@@ -13,6 +13,12 @@
       - setuptools
     state: latest
 
+- name: Downgrade setuptools if 2to3 is required
+  pip:
+    name: "setuptools<58"
+    virtualenv: "{{ python_venv_path }}"
+  when:  requires_python2to3
+
```

Comment:
would it not work to just pin setuptools lower in s&co's own `requirements.txt` file, and make sure it comes first?

Reply:
🤔 I was assuming that setuptools has to be installed first; but I should be able to test this pretty easily.

Reply:
just tested, they can't be specified in the same requirements file — the setuptools version needs to be downgraded before installing the package that needs it